### ISSUE TITLE
Write coordinates along normal mode

### ISSUE THREAD

**Issue by similt** [ajjackson/ascii-phonons#32]
When we displace the system along a normal mode, how to write the new coordinates in  POSCAR format?

**Comment by aronwalsh** [comment]
@similt I suggest https://github.com/JMSkelton/ModeMap for that purpose.

**Comment by ajjackson** [comment]
Right, if you can produce input to ascii-phonons then presumably you already have a way of generating displaced structures! (Probably Phonopy...)
